tests/test_detector.py

```diff
@@ -91,6 +93,16 @@ def addBadCameraSys(dw):
         with self.assertRaises(lsst.pex.exceptions.Exception):
             DetectorWrapper(modFunc=addBadCameraSys)
 
+        # These break in the pybind layer
+        for crosstalk in ([1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0],  # 1D and not numpy
+                          np.array([1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0]),  # 1D, wrong numpy type
+                          np.array([1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0], dtype=np.float32),  # 1D
+                          ):
+            self.assertRaises(TypeError, DetectorWrapper, crosstalk=crosstalk)
+        # This breaks in the Detector ctor: wrong shape
+        self.assertRaises(lsst.pex.exceptions.InvalidParameterError,
```

Comment:
Since there are two different `throw`s in `detector.cc`, you should probably have one test for each of those conditions.